front/lib/resources/data_source_view_resource.ts

```diff
@@ -353,6 +355,46 @@ export class DataSourceViewResource extends ResourceWithVault<DataSourceViewMode
     return dataSourceViews ?? null;
   }
 
+  static async search(
+    auth: Authenticator,
+    searchParams: {
+      [key: string]: string | number | undefined;
```

Comment:
that's too permissive, especially that they are directly injected in sequelize query - we should rather have the exact list of allowed parameters